src/scss/reviews.scss

```diff
@@ -0,0 +1,129 @@
+@import "variables";
```

Comment:
This import should be in index.sccc

Reply:
![image](https://user-images.githubusercontent.com/78299696/212132979-aa8ea9cb-3add-4540-ad29-577e918959dc.png)

Reply:
when removing import variables.scss in file where variables are used.

Reply:
This is normal behavior. You shouldn't import variables every time